pkgs/development/compilers/symbiflow-yosys-plugins/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv
+, fetchFromGitHub
+, symbiflow-yosys
+, zlib
+, readline
+}:
+
+stdenv.mkDerivation rec {
+  pname   = "symbiflow-yosys-plugins";
+  version = "1.0.0.7-0060-g7454cd6";
```

Comment:
The `g7454cd6` doesn't seem to match the value of `rev`. Where it came from?

Reply:
I seem to have miss-copied, I forgot the 8th hex digit.